_extras/guide.md

```diff
@@ -1,6 +1,24 @@
 ---
 title: "Instructor Notes"
 ---
-FIXME
+
+## Episode 3: Monitor the training process
+When episode 3 is taught on a different day then episode 2, it is very useful to start with a recap of episode 2. This will help learners in the big exercise on creating a neural network.
```

Comment:
Would the key points of episode 2 be a good reference to check learner progress?

Reply:
I think the most useful form of recap is going through the code of episode 2. But the key points are probably also useful to start with.